ShowPot/ShowPot/Common/UI/Protocol/ReusableCell.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  ReusableCell.swift
+//  ShowPot
+//
+//  Created by Daegeon Choi on 7/17/24.
+//
+
+import Foundation
+import UIKit
```

Comment:
```suggestion
import UIKit
```
요부분만 수정하고 병합하면 될꺼같은데요 ?? 너무 수고많으셨습니다 !!!